test/Microsoft.ML.Tests/DatabaseLoaderTests.cs

```diff
@@ -186,7 +186,7 @@ public void IrisVectorLightGbmWithLoadColumnName()
             }).PredictedLabel);
         }
 
-        [Fact]
+        [X86X64FactAttribute("Onlys supports x86/x64 architectures.")]
```

Comment:
type-o `Onlys supports`

What about this test only supports x86/x64 architectures? We should explain in the string.

Reply:
Added in a better explanation.